src/applications/simple-forms/form-upload/components/CustomReviewTopContent.jsx

```diff
@@ -53,6 +60,25 @@ const CustomReviewTopContent = () => {
     </div>
   );
 
+  const formattedPhoneNumber = () =>
+    `(${phoneNumber.substr(0, 3)}) ${phoneNumber.substr(
```

Comment:
I think phoneNumber can have dashes in it. Is that right? if so we can replace those out.

Reply:
Good point! Fixed.